components/brave_wallet_ui/components/desktop/popup-modals/transaction_details_modal/transaction_details_modal.style.ts

```diff
@@ -197,7 +197,6 @@ export const LoadingIcon = styled(ProgressRing)<{
   status: BraveWallet.TransactionStatus
 }>`
   --leo-progressring-size: 14px;
-  --leo-progressring-background-color: rgba(255, 255, 255, 0.4);
```

Comment:
cc @Douglashdaniel I had a talk with design and they've updated the progress ring to always use the progress ring color with `30%` opacity as the background color for the ring (and removed this CSS variable).

I don't think this should break anything, but wanted to get your eyes on this before we land.